inksdk/inksample/src/main/java/com/microsoft/device/inksample/MainActivity.kt

```diff
@@ -82,36 +85,49 @@ class MainActivity : AppCompatActivity() {
         }
     }
 
-    fun clickClear(@Suppress("UNUSED_PARAMETER") view: View) {
+    private fun setupClickListeners() {
+        findViewById<Button>(R.id.btnClear)?.setOnClickListener(::clickClear)
```

Comment:
You could consider using `ViewVindings` https://developer.android.com/topic/libraries/view-binding

Reply:
Definitely! I wanted to make the smallest change possible here since it was not the scope of the PR (only fixed Github actions build errors) and we'll have a separate one just for using view-binding everywhere:)